shared/fs/browser/index.stories.tsx

```diff
@@ -100,18 +95,6 @@ export default () => {
         />
       </Kb.Box2>
     ))
-    .add('in conflict', () => (
```

Comment:
why did you take away this story?

Reply:
because the conflict state isn't passed in from the browser. There's one in the banner stories.